### PR TITLE
Grades from Miri (revised)

### PR DESCRIPTION
**Pull request by miribar**
Hi @ameedf :-)
Please review my revised version to grades.
One problem i already see with my revised version - I should've placed the 'grade' param declaration outside the loop.
Thanks!

### CODE REVIEW

File: Grades.java [hunk 1]
```diff
@@ -13,6 +15,55 @@
...
+            System.out.println("The number of students must be positive.");
+            return;
+        }
+
+        byte studentsArray[] = new byte[numOfStudents];
+        short grade = 0;
+        byte indexHighest = 0;
+        byte indexLowest = 0;
+        byte highest = 0;
+        byte lowest = 100;
```
Comment (ameedf) [inline Grades.java]: Do you really need the highest and the lowest VALUES ?
Index is enough.. isn't it ?

Comment (miribar) [reply Grades.java]: @ameedf Getting the highest and lowest values was a part of the assignment..

Comment (ameedf) [reply Grades.java]: indeed.. but my question is: isn't knowing the index is enough ? why do you have to keep (in memory) these variables ? isn't it a "waste" of memory ? 
I mean whenever you wrote 'highest' you could've wrote 'myArray[ indexHighest ]' :)

Comment (miribar) [reply Grades.java]: You are totally correct! I could do that :-)

**Comment by ameedf** [comment]
💯 as always :)

**Pull request closed without merging**